rever/activities/travis_ci.xsh

```diff
@@ -0,0 +1,99 @@
+"""Activity for keeping a changelog from news entries."""
+import os
+import re
+import sys
+
+from xonsh.tools import print_color
+
+from rever import vcsutils
+from rever.activity import Activity
+from rever.tools import eval_version, replace_in_file
+
+TRAVIS_CI = """sudo: False
+
+language: python
+
+matrix:
+  include:
+    - python: 3.6
+
+install:
+  # Install conda
+  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - conda config --set always_yes yes --set changeps1 no
+  - conda config --add channels conda-forge
+  - conda update conda
+
+  # Install dependencies
+  - conda install {conda_deps} -c (conda_channels)
+  - pip install {pip_deps}
+  - python setup.py install
+
+# script:
+  # - set -e
+  # - # Command to build your docs
+  # - pip install doctr
+  # - cd docs
+  # - make html
+  # - cd ..
+  # - doctr deploy . --deploy-repo ergs/rever-docs --built-docs ./docs/_build/html
+
+notifications:
+  email: false
+
+"""
```

Comment:
I don't think that this is anywhere near general enough. There is nothing in rever that states the project has to be python oriented. I suggest taking a look at how the Docker containers are built. Basically the user provides depenendcy lists (for different potential package managers) and an install command / script. For CI, we'd want a test line/script as well.  You may also want a flag for enabling doctr, since that seems to be here too.

Reply:
Hmm, that's fair (I was the one doing the copy-paste from rever's travis).